plugins/cisco_firepower_management_center/icon_cisco_firepower_management_center/util/api.py

```diff
@@ -76,6 +76,30 @@ def update_address_group(self, payload):
             json_data=payload,
         )
 
+    def post_policy(self, payload: dict):
+        return self._call_api(
+            "POST", f"fmc_config/v1/domain/{self.domain_uuid}/policy/accesspolicies", json_data=payload
+        )
+
+    def get_policies(self, policy_id: str = None):
+        if policy_id:
+            return self._call_api("GET", f"fmc_config/v1/domain/{self.domain_uuid}/policy/accesspolicies/{policy_id}")
+        else:
+            return self._call_api("GET", f"fmc_config/v1/domain/{self.domain_uuid}/policy/accesspolicies")
```

Comment:
Shouldn't this be handled with `run_with_pages`?

Reply:
Great suggestion! 😄